docs/file-formats/xml-data-file.md

```diff
@@ -20,11 +20,11 @@ The import file must start with the standard XML header:
 
 The `<knora>` element describes all resources that should be imported. It has the following attributes:
 
-- `xmlns`: `"https://dasch.swiss/schema"` (required)
-- `xmlns:xsi`: `"http://www.w3.org/2001/XMLSchema-instance"` (required)
-- `xsi:schemaLocation`: `"https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_toolresources/schema/data.xsd"` (required)
-- `shortcode`: project shortcode, e.g. "0801" (required)
-- `default-ontology`: name of the ontology (required)
+- `xmlns`: XML namespace identifier (required, always the same)
+- `xmlns:xsi`: URL to the XML Schema instance namespace of W3C (required, always the same)
+- `xsi:schemaLocation`: URL to the DSP XML schema, located in the DSP-TOOLS GitHub repository (required, always the same)
```

Comment:
The links are still available some lines further down (but I assume you didn't see them because GitHub diff doesn't show them to you, as those lines are untouched).